nrt/data/DynaFlow/cases.py

```diff
@@ -316,3 +316,43 @@
 job_file = os.path.join(os.path.dirname(__file__),  "StaticVarCompensators", "SVarC_StepURef", "SVarCPVRemoteModeHandling", "SVarC.jobs")
 
 test_cases.append((case_name, case_description, job_file, 30, standardReturnCodeType, standardReturnCode))
+
+##########################################
```

Comment:
Everything can be removed here. As we don't add any nrt, the cases.py file must not be changed.